wolfcrypt/src/sp_int.c

```diff
@@ -2316,7 +2319,7 @@ int sp_prime_is_prime_ex(sp_int* a, int t, int* result, WC_RNG* rng)
 #endif
 
     if (a == NULL || result == NULL || rng == NULL)
-        err = MP_VAL;
+        return MP_VAL;
```

Comment:
Prefer that the next if statement has err check.

Reply:
OK